### PR TITLE
Add database client to connect to the db.  

### PR DESCRIPTION
**Pull request by ryan-endacott**
This adds a database client that connects to rethink.  It also contains tests, but I will probably need to update the Travis CI config to install rethinkdb to run the tests.  

Depends on #19.

### CODE REVIEW

**Comment by ryan-endacott** [comment]
Woot!  Finally got rethinkdb working on Travis.  So the tests pass, and this is ready for review.

File: src/app/models/user/user.go [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package user
 
-import (
-	"github.com/EaseApp/web-backend/src/config/db"
+/*
```
Comment (Felttrip) [inline src/app/models/user/user.go]: Are we planing on using this logic in the model later?

File: src/db/db_test.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package db
```
Comment (Felttrip) [inline src/db/db_test.go]: :+1+ on the tests

**Comment by Felttrip** [comment]
@ryan-endacott looks good. LGTM.

File: src/app/models/user/user.go (hunk 1 above)
Comment (ryan-endacott) [inline src/app/models/user/user.go]: Yeah we will. Just removed it for now.

Comment (ryan-endacott) [inline src/app/models/user/user.go]: For usage, I plan on having the models have a db client as a dependency, then use that for db access.

File: src/db/db_test.go (hunk 2 above)
Comment (ryan-endacott) [inline src/db/db_test.go]: Thanks!

**Comment by ryan-endacott** [comment]
Thanks for review!

**Pull request merged**